### PR TITLE
[feat] create new proposal

### PR DESCRIPTION
**Pull request by greatdolphinls**

### CODE REVIEW

File: containers/Vote/NewProposal/ProposalSubForm/index.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    justifyContent: 'center',
+    padding: theme.spacing(1.5, 2.5),
+  },
+  header: {
+    display: 'flex',
+    justifyContent: 'space-between',
+  }
+}))
+
+const PERIOD_TYPES = [
```
Comment (bigwampa) [inline containers/Vote/NewProposal/ProposalSubForm/index.js:25]: This should go to 30 days

Comment (bigwampa) [reply containers/Vote/NewProposal/ProposalSubForm/index.js:25]: "days" should be omitted from the options and just included in the field description.

Comment (bigwampa) [reply containers/Vote/NewProposal/ProposalSubForm/index.js:25]: 1 and 2 are not options. 3 is the minimum

**Review by bigwampa — DISMISSED** [review]

File: containers/Vote/NewProposal/ProposalSubForm/index.js (hunk 1 above)
Comment (greatdolphinls) [reply containers/Vote/NewProposal/ProposalSubForm/index.js:25]: How about using number input instead of select?

**Review by bigwampa — APPROVED** [review]

**Pull request merged**